### DISCUSSION TITLE
idea: share some additional snapshoting guides

### DISCUSSION THREAD

**Discussion by playmobilmeister** [SpiralLinux/SpiralLinux-project#475] [General]
Happy Spiral user on two laptops here. 
I'm looking out for some guides on automatic snapshoting tricks. Can you share some links to good ones that you know of somewhere, please?
Specifically, I was thinking of working on the following and sharing the results:

1. Add to the snapshot description some more details in case of kernel updates, xfce updates and other essential stuff like it. 

2. Manually changing anything via apt, apt-get, aptitude would trigger a question "Do you want to add a more detailed description to the newly created snapper description?

Also, how can anyone contribute to the project, aside for a good review in distrowatch?

**Comment by geckolinux** [comment]
Hi there, really glad to hear that SpiralLinux is working for you.

Thanks for the good ideas about the automatic snapshot descriptions. (I converted your issue report into a discussion to give it a bit more visibility and so that it doesn't get closed like it would as an issue.) It probably won't be possible to implement that by default in SpiralLinux because the automatic "apt" label on snapshots is implemented in the upstream Debian `snapper` [package](https://packages.debian.org/bookworm/amd64/snapper/filelist), specifically in `/etc/apt/apt.conf.d/80snapper` , and SpiralLinux doesn't modify Debian packages.

Here are some references about how the whole thing works:
- https://manpages.debian.org/bookworm/apt/apt.conf.5.en.html
- https://wiki.debian.org/AptConfiguration
- http://snapper.io/manpages/snapper.html
- https://doc.opensuse.org/documentation/leap/reference/html/book-reference/cha-snapper.html

**Reply by playmobilmeister** [reply]
Thanks. What about contributions in the form of documenting, money or maybe some other stuff that is needed?

**Reply by geckolinux** [reply]
Hi there, really appreciate you asking. I don't really have anything set up for receiving donations. Documentation, HOWTOs, and answering other users' questions is greatly appreciated.
